common/djangoapps/util/milestones_helpers.py

```diff
@@ -82,6 +82,15 @@ def remove_prerequisite_course(course_key, milestone):
     )
 
 
+def remove_course_references(course_key):
+    """
+    Remove course milestones and course content milestones that are referenced to specified course_key
```

Comment:
Docstring can't be more than 80 chars.
Also please add the dot at the ant of the string.

Reply:
Miss this point (80 chars for docstrings), thanks.